impl/src/test/java/org/pentaho/di/trans/dataservice/streaming/execution/StreamingGeneratedTransExecutionTest.java

```diff
@@ -125,9 +124,6 @@ public void setup() throws Exception {
 
   @Test
   public void testRunCachedWindow() throws Exception {
-    when( streamExecutionListener.getCachePreWindow() ).thenReturn( rowIterator );
-    when( genTrans.isRunning() ).thenReturn( true );
```

Comment:
this `when( genTrans.isRunning() ).thenReturn( true );` shows up a lot, but never gets hit. The methods where this is intended to be hit look at instance variables and don't actually use that method,.